packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-services/client/stage0-client-service.ts

```diff
@@ -270,6 +292,20 @@ export class Stage0ClientService extends SATPService {
       getHash(preSATPTransferRequest),
     );
 
+    await storeLog(
+      {
+        sessionID: sessionData.id,
+        type: "preSATPTransferRequest",
```

Comment:
You should use the message type enum from the protos

Reply:
I have added the enum types, but I'm unsure if they correctly correspond to the appropriate client and server roles that will be stored in the log.